src/sizeguide/SizeGuideProductInfo.jsx

```diff
@@ -1,12 +1,34 @@
-import React from "react";
 import PropTypes from "prop-types";
-import DetailSection from "./DetailSection.jsx";
-import SizeSelector from "../api/SizeSelector";
+import React from "react";
 import { withTranslation } from "react-i18next";
-import HoverContainer from "./HoverContainer.jsx";
+import SizeSelector from "../api/SizeSelector";
+import uiOptions from "../api/uiOptions";
 import CookieHideWrapper, { hideSizeMe } from "../common/CookieHideWrapper.jsx";
 import { openLoginFrame } from "../common/LoginFrame";
-import uiOptions from "../api/uiOptions";
+import DetailSection from "./DetailSection.jsx";
+import HoverContainer from "./HoverContainer.jsx";
+
+/*const inchFractionOptions = {
+    0: "",
+    1: "1/8",
+    2: "1/4",
+    3: "3/8",
+    4: "1/2",
+    5: "5/8",
+    6: "3/4",
+    7: "7/8",
+};*/
```

Comment:
ja nää vanhat roiskeetkin voi aina huoletta poistaa, ne löytyy sitten gitistä jos pitää palata taaksepäin